pkg/odo/cli/url/create.go

```diff
@@ -99,7 +99,7 @@ func (o *URLCreateOptions) Run() (err error) {
 		return errors.Wrapf(err, "failed to persist the component settings to config file")
 	}
 	log.Successf("URL %s created for component: %v\n", o.urlName, o.Component())
-	log.Info("To create URL on the OpenShift Cluster, please use `odo push`")
+	log.Italic("\nTo create URL on the OpenShift Cluster, please use `odo push`")
```

Comment:
Please remove the new line, it's adding two newlines in the final output

```
[mrinaldas@localhost ~]$ odo url create example
 ✓  URL example created for component: nodejs-mrinaldas-axlj


To create URL on the OpenShift Cluster, please use `odo push`
```

Reply:
It's because of the previous success message having a newline :( I removed it from line 101.